tests/test_config.py

```diff
@@ -1,14 +1,16 @@
 '''
 Set of test for loading YAML files using the function call load_yaml
 '''
-#pylint: disable=unused-variable
+#pylint: disable=unused-variable, unspecified-encoding
```

Comment:
Is there a reason we can't or shouldn't use `encoding="utf-8"` in the open statement? In general, it's good to fall in line with what the linter wants. There are of course exceptions.

Reply:
Ah thats very fair, I'll change over to that